config/initializers/elasticsearch.rb

```diff
@@ -1,4 +1,4 @@
-yaml = Rails.application.config_for(:elasticsearch)
+yaml = YAML.load_file("#{Rails.root}/config/elasticsearch.yml").presence
```

Comment:
@lsamuels-fearless , I added the `.presence` method to ensure that if no configuration values are found, the `yaml` will be set to `nil` rather than an empty hash. That means that in the future, a error will be raised if we try to access those missing config values during a deploy.